src/javascript/_common/url.js

```diff
@@ -126,7 +126,12 @@ const Url = (() => {
         return `${origin}/${path}${pars ? `?${pars}` : ''}`;
     };
 
-    const getAllowedLocalStorageOrigin = (is_traders_hub) => {
+    const urlForWalletAccount = (path, pars) => {
+        const origin = getAllowedLocalStorageOrigin(true, true) || deriv_app_domain;
+        return `${origin}/${path}${pars ? `?${pars}` : ''}`;
+    };
+
+    const getAllowedLocalStorageOrigin = (is_traders_hub, is_wallet = false) => {
```

Comment:
is the additional param actually required, we can reuse `is_traders_hub` right, just change the name to better name. If removing is_wallet then ignore the 2 comments which is given below.